src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php

```diff
@@ -24,7 +24,7 @@
 class YamlLintCommand extends BaseLintCommand
 {
     protected static $defaultName = 'lint:yaml';
-    protected static $defaultDescription = 'Lints a file and outputs encountered errors';
+    protected static $defaultDescription = 'Lints a YAML file and outputs encountered errors';
```

Comment:
I think the base lint commandalso needs the same patch, isn't it? (same for xliff)

Reply:
good catch, I changed them too

Reply:
same for the twig one :)

Reply:
already done in same push ;)